operations/helm/charts/grafana-agent/README.md

```diff
@@ -159,6 +159,11 @@ components like [discovery.kubernetes][] to work properly.
 
 [discovery.kubernetes]: https://grafana.com/docs/agent/latest/flow/reference/components/discovery.kubernetes/
 
+### controller.autoscaling
+
+`controller.autoscaling.enabled` enables the creation of a HorizontalPodAutoscaler for `controller.type` deployment. The use case for `controller.autoscaling` is an [app agent receiver](https://grafana.com/docs/agent/latest/configuration/integrations/integrations-next/app-agent-receiver-config/) configured Grafana Agent.
```

Comment:
```suggestion
`controller.autoscaling.enabled` enables the creation of a HorizontalPodAutoscaler. It is only used when `controller.type` is set to `deployment`.

`controller.autoscaling` is intended to be used with an [app agent receiver](https://grafana.com/docs/agent/latest/configuration/integrations/integrations-next/app-agent-receiver-config/) configured Grafana Agent.
```

Reply:
(You'll also have to make this change in the readme template since the README is also autogenerated)